src/component/RoadmapCanvas.js

```diff
@@ -3,101 +3,134 @@ import ContextMenu from "d3-v4-contextmenu";
 
 export class RoadmapCanvas {
   type;
-
   targetElement;
 
+  style = {
+    barHeight: 20,
+    topPadding: 20
+  };
+
+  /**
+   * @param {object} options
+   */
   constructor(options) {
     this.type = options.type;
     this.targetElement = d3.select("#" + options.target);
+    this._setStyle(options.style !== undefined ? options.style : {});
+  }
+
+  /**
+   * @param {object} style
+   * @private
+   */
+  _setStyle(style) {
+    if (style.barHeight !== undefined) {
+      this.style.barHeight = parseInt(style.barHeight);
+    }
+    if (style.topPadding !== undefined) {
+      this.style.topPadding = parseInt(style.topPadding);
+    }
+    this.style.gap = this.style.barHeight + 4;
   }
 
+  /**
+   * @param {Roadmap} roadmap
+   */
   render(roadmap) {
-    if (this.type === "taks") {
-      this._render(roadmap.getTasks(), roadmap);
-    } else if (this.type === "people") {
-      this._render(roadmap.getPeople(), roadmap);
+    if (this.type === "story") {
+      this._renderStoryTask(roadmap);
+    } else if (this.type === "assignee") {
+      this._renderAssigneeTask(roadmap);
     } else {
-      this._render(roadmap.getTasks(), roadmap);
-      this._render(roadmap.getPeople(), roadmap);
+      this._renderStoryTask(roadmap);
+      this._renderAssigneeTask(roadmap);
     }
+
   }
 
-  _render(items, roadmap) {
-    const barHeight = 20;
-    const gap = barHeight + 4;
-    const topPadding = 20;
+  /**
+   * @param {Roadmap} roadmap
+   * @private
+   */
+  _renderStoryTask(roadmap) {
```

Comment:
`_renderStoryTask` and `_renderAssigneeTask` have duplicate codes...
But, I can't think of any solution